packages/expo-av/android/src/main/java/expo/modules/av/AVManager.java

```diff
@@ -181,10 +184,26 @@ public void onHostPause() {
 
   @Override
   public void onHostDestroy() {
-    mContext.unregisterReceiver(mNoisyAudioStreamReceiver);
-    for (final Integer key : mSoundMap.keySet()) {
-      removeSoundForKey(key);
+    if (isRegistered) {
+      try {
+        mContext.unregisterReceiver(mNoisyAudioStreamReceiver);
+        isRegistered = false;
+      } catch (final Exception exception) {
```

Comment:
I don't think we should ignore errors here. Currently, code should not throw an exception. If it does, we want to know that in order ton fix it.

Reply:
OK, I won't ignore exceptions here.